src/d3a/d3a_core/sim_results/endpoint_buffer.py

```diff
@@ -91,7 +90,8 @@ def generate_json_report(self):
             "bills": self.bills,
             "tree_summary": self.tree_summary,
             "status": self.status,
-            "device_statistics": self.device_statistics_time_str_dict,
+            "device_statistics": self.device_statistics.device_stats_time_str,
+            "device_statistics_redis": self.device_statistics.flat_results_time_str,
```

Comment:
Do we really want to have both device statistics in the local export or is this a remnant?

Reply:
Oops, this is a remnant, fixing this.